### PR TITLE
[4.0] TinyMCE toolbar mode

### PR DESCRIPTION
**Pull request by brianteeman**
As requested #32090

There are now 4 modes for the toolbar.

### Sliding (default)
sliding

### Wrap (old style)
image

### Floating
float

### Scrolling (no idea why but its the 4th option)
scrolling

They are configured **per set** in the plugin settings

### Documentation required
Yes: the contents of this post can be added to the docs site

### CODE REVIEW

**Comment by drmenzelit** [comment]
Great :-) Will test asap

File: administrator/language/en-GB/plg_editors_tinymce.ini [hunk 1]
```diff
@@ -50,22 +50,27 @@ PLG_TINY_FIELD_SKIN_ADMIN_LABEL="Administrator Skin"
...
 PLG_TINY_FIELD_VALUE_NUMERIC="numeric"
 PLG_TINY_FIELD_VALUE_P="P Elements"
 PLG_TINY_FIELD_VALUE_RAW="raw"
 PLG_TINY_FIELD_VALUE_RELATIVE="Relative"
 PLG_TINY_FIELD_VALUE_RTL="Right to Left"
+PLG_TINY_FIELD_VALUE_SCROLLING="Scrolling"
+PLG_TINY_FIELD_VALUE_SLIDING="Sliding"
 PLG_TINY_FIELD_VALUE_SIMPLE="Simple"
 PLG_TINY_FIELD_WORDCOUNT_LABEL="Word Count"
+PLG_TINY_FIELD_VALUE_WRAP="Wrap"
```
Comment (Quy) [inline administrator/language/en-GB/plg_editors_tinymce.ini]:
```suggestion
PLG_TINY_FIELD_VALUE_SIMPLE="Simple"
PLG_TINY_FIELD_VALUE_WRAP="Wrap"
PLG_TINY_FIELD_WORDCOUNT_LABEL="Word Count"
```

**Comment by drmenzelit** [comment]
I have tested this item :white_check_mark: successfully on 3d87713421f86823ff802aa34c58ce204b950eb8<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32091">issues.joomla.org/tracker/joomla-cms/32091</a>.</sub>

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 3d87713421f86823ff802aa34c58ce204b950eb8<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32091">issues.joomla.org/tracker/joomla-cms/32091</a>.</sub>

**Comment by infograf768** [comment]
rtc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32091">issues.joomla.org/tracker/joomla-cms/32091</a>.</sub>

**Comment by chmst** [comment]
Thanks

**Comment by brianteeman** [comment]
thx

**Pull request merged**